src/frontend/komponenter/Fagsak/useFagsakApi.ts

```diff
@@ -116,9 +119,39 @@ const useFagsakApi = (
                 ? BehandlingResultat.OPPHØRT
                 : context.behandlingResultat;
 
+        const mapTilPeriodeResultater = (
+            samletVilkårResultat: IVilkårResultat[]
+        ): IRestPeriodeResultat[] => {
+            const identResultater: Map<string, IRestVilkårResultat[]> = new Map();
+            samletVilkårResultat.map(resultat => {
```

Comment:
Hvis du typer resultat bør du kunne unngå ts-ignore. Også burde man kanskje lage "temp" objekter for typesikkerhet.